corehq/apps/app_manager/util.py

```diff
@@ -670,8 +670,10 @@ def get_form_source_download_url(xform):
     if not xform.build_id:
         return None
 
-    from corehq.apps.app_manager.models import Application
-    app = Application.get(xform.build_id)
+    app = get_app(xform.domain, xform.build_id)
+    if app.is_remote_app():
```

Comment:
Do we still have remote apps? I thought those were killed long ago?

Reply:
We don't support creating new ones but still have some references to old ones: https://bitbucket.org/ctsims/commcare-pact/src/8f1079941fbd4d34e965fed03ca369f2bb6040e1/pact-app/twooh/?at=default

Considering this error happened on a form that was received this month I think it's safe to say they are still using the remote app version.

I couldn't open the app because of this: https://github.com/dimagi/commcare-hq/pull/20943